site/source/docs/blockchain_configuration.md

```diff
@@ -34,7 +34,7 @@ module.exports = {
 }
 ```
 
-Similar to [configuring Smart Contracts](/docs/contracts_configuration.html), this config contains environments that help configuring certain parameters differently depending of the environment. You can read more about [environments here](https://embark.status.im/docs/environments.html).
+Similar to [configuring Smart Contracts](/docs/contracts_configuration.html), this config contains environments that help configuring certain parameters differently depending of the environment. You can read more about [environments here](https://embark.embarklabs.io/docs/environments.html).
```

Comment:
Might be a good idea to take the opportunity and make it an "absolute relative" link like `/docs/environments.html` so it work "offline" as well.